end-to-end-test/local/specs/namespace-columns-utils.js

```diff
@@ -0,0 +1,40 @@
+function clickColumnSelectionButton(patientCnaTable) {
+    $(`[data-test=${patientCnaTable}]`)
+        .$('button*=Columns')
+        .click();
+}
+
+function selectColumn(namespaceColumn1) {
+    $(`[data-id="${namespaceColumn1}"]`).click();
+}
+
+waitForTable = table => {
```

Comment:
you should always use ```const```
the risk is that something becomes unintentionally global.

Reply:
Fixed in https://github.com/cBioPortal/cbioportal-frontend/pull/4443